### PR TITLE
Add Kyverno updaterequests alert and tests

### PR DESCRIPTION
**Pull request by fhielpos**
Closes: https://github.com/giantswarm/giantswarm/issues/28807
Signed-off-by: Franco <franco@giantswarm.io>

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml [hunk 1]
```diff
@@ -42,6 +42,21 @@ spec:
         severity: page
         team: shield
         topic: kyverno
+    - alert: KyvernoUpdateRequestsCountTooHigh
+      annotations:
+        description: "{{`Kyverno {{ $labels.kind }} are too high. This is an indicator that Kyverno\'s background controller may not be able to create some resources.`}}"
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml:47]: Why did you add a \ here ? Otherwise LGTM

**Review by QuentinBisson — APPROVED** [review]

File: helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml (hunk 1 above)
Comment (fhielpos) [reply helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml:47]: Not sure, I copied it from another rule, will try without it
